pom.xml

```diff
@@ -875,15 +875,21 @@
                       com.puppycrawl.tools.checkstyle.checks.javadoc.JavadocMethodCheck.RegularClass
                     </exclude>
                     <!-- Swing related classes -->
-                    <exclude>com.puppycrawl.tools.checkstyle.gui.BaseCellEditor*</exclude>
+                    <exclude>com.puppycrawl.tools.checkstyle.gui.BaseCellEditor</exclude>
```

Comment:
Can we remove all classes in `gui`  from excludes in this PR?

Reply:
I am confused why these excludes are changing to only remove the `*` on the end? I believe these stars to are include any subclasses, so we are changing it to account for the subclasses but ignore the primary classes? Shouldn't these excludes be fully removed and we will keep the individual coverage on the bottom?

Reply:
That is exactly what I did. `TreeTable` and `TreeTable.TreeTableCellEditor` have their own coverage config now.
Same for `TreeTableModelAdapter` and `TreeTableModelAdapter.UpdatingTreeModelListener`.

All other classes do not have inner classes. For them, `*` has no effect.

Reply:
@pbludov my mistake, I thought that once we defined coverage rules elsewhere, that we did not need thees excludes anymore. resolved.